packages/curve-common/src/widgets/Header/HeaderLogo.tsx

```diff
@@ -26,17 +26,17 @@ export const HeaderLogo = ({ appName, isLite, sx }: HeaderLogoProps) => (
     <Image src={LogoImageSrc} alt="Curve" width={30} />
     {appName ? (
       <Box display="inline-flex" flexDirection="column">
-        <Typography variant="headingSBold" sx={{ lineHeight: '1.2rem', textTransform: 'none' }} color="textPrimary">
+        <Typography variant="headingSBold" sx={{ marginBottom: 0, lineHeight: '1.2rem', textTransform: 'none' }} color="textPrimary">
```

Comment:
Wait are these marginBottoms necessary? Quickly inspecting this on curve.fi shows me margin is already set to 0px for these typographies.

Reply:
Yeah I can remove this. There was a version of typography with marginBottom but that was fixed, so this can go.